# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/pkg_patch/pkg_gcc.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -ruN gcc-3.4.4.orig/libiberty/regex.c gcc-3.4.4/libiberty/regex.c
--- gcc-3.4.4.orig/libiberty/regex.c	2003-04-15 16:24:26.000000000 +0200
+++ gcc-3.4.4/libiberty/regex.c	2005-09-15 16:29:40.000000000 +0200
@@ -302,7 +302,7 @@
 
 /* Integer type for pointers.  */
 # if !defined _LIBC && !defined HAVE_UINTPTR_T
-typedef unsigned long int uintptr_t;
+/*typedef unsigned long int uintptr_t;*/ 
 # endif
 
 /* Should we use malloc or alloca?  If REGEX_MALLOC is not defined, we
